assets/base/i18n/es.json

```diff
@@ -585,6 +585,7 @@
     "settings.save": "Guardar",
     "smobile.search.recent_title": "Búsquedas recientes en {teamName}",
     "snack.bar.favorited.channel": "Este canal estaba como favorito",
+    "snack.bar.info.copied": "El info ha sido copiado al portapapeles",
```

Comment:
Thank you for adding the translations. Nevertheless, they are handled by a different system, so it is better for us if you only add the English string. Same for the brazilian translation.

Reply:
Didn't know about this, thank you for explaining!